### PR TITLE
chore(translation): migrate code from googleapis/python-translate

### PR DESCRIPTION
**Pull request by telpirion**
- Add new "quickstart" samples [(#547)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/547)
- Quickstart tests [(#569)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/569)
- Add translate samples [(#574)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/574)
- Generate most non-appengine readmes
- Update requirements.txt
- added model in the sample [(#650)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/650)
- added with and without model separately [(#657)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/657)
- Translate API no longer requires an API key. [(#659)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/659)
- Auto-update dependencies. [(#715)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/715)
- Changes default model to premium [(#749)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/749)
- Updates readme template to show credential auth instead of api key. [(#802)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/802)
- Updates translate version to match speech and language. [(#843)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/843)
- Translate update [(#886)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/886)
- Remove cloud config fixture [(#887)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/887)
- Unicode for translation
- Adds check for translated text
- Unicode for translation
- Unicode for translation
- Unicode for translation
- missing newline
- Adds six for 2/3 compatibility
- Changes encoding style.
- Fix unicode
- renaming of the product to Google Translation API [(#910)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/910)
- Update translate readme
- Re-generate all readmes
- Fix README rst links [(#962)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/962)
- Auto-update dependencies. [(#1004)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1004)
- Auto-update dependencies. [(#1055)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1055)
- Auto-update dependencies. [(#1093)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1093)
- Update all generated readme auth instructions [(#1121)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1121)
- Added Link to Python Setup Guide [(#1158)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1158)
- Auto-update dependencies. [(#1186)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1186)
- Fixed failed tests on Kokoro (Spanner + Translate) [(#1192)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1192)
- Added "Open in Cloud Shell" buttons to README files [(#1254)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1254)
- Auto-update dependencies. [(#1377)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1377)
- Regenerate the README files and fix the Open in Cloud Shell link for some samples [(#1441)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1441)
- Update READMEs to fix numbering and add git clone [(#1464)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1464)
- Add translate region tags [(#1488)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1488)
- Include the comma in the translation [(#1787)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1787)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- Translation v3beta1 samples [(#2084)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2084)
- fixing translate-with-glossary bug [(#2323)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2323)
- Translate beta samples fix [(#2327)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2327)
- Using glossaries with tts and vision tutorial sample code [(#2325)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2325)
- fix: refactored MP3 file creation test for Hybrid glossaries samples [(#2379)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2379)
- Fix variable names in comments [(#2400)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2400)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Update v2 samples to explicitly use v2 library [(#2498)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2498)
- fix: translate test [(#2671)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2671)
- automl: add natural language sentiment analysis ga samples [(#2677)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2677)
- fix region tag typo [(#2731)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2731)
- Migrate published samples [(#2759)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2759)
- translate v3 snippets [(#2745)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2745)
- Translate: migrate published samples [(#2768)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2768)
- Translate: migrate published glossaries samples [(#2769)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2769)
- translate: increase timeout [(#2937)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2937)
- Translate: migrate published  v3 translate batch samples [(#2914)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2914)
- chore(deps): update dependency google-cloud-storage to v1.26.0 [(#3046)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3046)
- chore(deps): update dependency google-cloud-translate to v1.7.0 [(#3084)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3084)
- translate: fix tests [(#3087)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3087)
- translate-v3:  samples [(#3034)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3034)
- chore(deps): update dependency google-cloud-vision to v0.42.0 [(#3170)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3170)
- chore(deps): update dependency google-cloud-texttospeech to v1 [(#3210)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3210)
- chore(deps): update dependency google-cloud-translate to v2 [(#3211)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3211)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- Update dependency google-cloud-vision to v1 [(#3227)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3227)
- translate: make test generic [(#3386)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3386)
- [translate] fix: mark some tests as flaky [(#3495)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3495)
- testing: replace @flaky with @pytest.mark.flaky [(#3496)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3496)
- chore(deps): update dependency google-cloud-storage to v1.28.0 [(#3260)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3260)
- translate: fix glossary leak issue [(#3572)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3572)
- chore: some lint fixes [(#3751)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3751)
- chore(deps): update dependency google-cloud-storage to v1.28.1 [(#3785)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3785)
- [translate] fix hybrid glossaries tests [(#3936)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3936)
- testing: start using btlr [(#3959)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3959)
- Replace GCLOUD_PROJECT with GOOGLE_CLOUD_PROJECT. [(#4022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4022)
- remove whitelist replace with allowlist [(#4050)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4050)
- chore(deps): update dependency google-cloud-storage to v1.29.0 [(#4040)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4040)
- chore(deps): update dependency google-cloud-texttospeech to v2.1.0 [(#4147)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4147)
- testing(translate): parameterize the timeout [(#4247)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4247)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- Update dependency flaky to v3.7.0 [(#4300)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4300)
- testing(translate): bump the timeout for operations [(#4258)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4258)
- chore: update templates
- chore(deps): update dependency google-cloud-storage to v1.30.0 (#26)
- feat!: migrate API to microgenerator (#31)
- chore: applies new samples README template
- chore(deps): update dependency google-cloud-translate to v3 (#37)
- test(samples): change string for utf8 translation (#48)
- samples: fixed flaky test (#57)
- automl beta [(#1575)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1575)
- remove translate prediction fallback [(#1598)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1598)
- skip automl model create/delete test [(#1608)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1608)
- Auto-update dependencies. [(#1658)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1658)
- Update AutoML region tags to use standard product prefixes [(#1669)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1669)
- Fix AutoML region tag typos [(#1687)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1687)
- Fixed name of model [(#1779)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1779)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- Updated beta version of automl [(#2124)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2124)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Auto-update dependencies. [(#2005)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2005)
- Translate: migrate published  v3 translate batch samples [(#2914)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2914)
- Update dependency google-cloud-automl to v0.10.0 [(#3033)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3033)
- chore(deps): update dependency google-cloud-storage to v1.26.0 [(#3046)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3046)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- chore(deps): update dependency google-cloud-storage to v1.28.0 [(#3260)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3260)
- chore: some lint fixes [(#3751)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3751)
- chore(deps): update dependency google-cloud-storage to v1.28.1 [(#3785)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3785)
- testing: start using btlr [(#3959)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3959)
- Replace GCLOUD_PROJECT with GOOGLE_CLOUD_PROJECT. [(#4022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4022)
- chore(deps): update dependency google-cloud-storage to v1.29.0 [(#4040)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4040)
- chore(deps): update dependency google-cloud-automl to v1 [(#4127)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4127)
- fix(translate): fix a broken test [(#4360)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4360)
- chore: remove automl samples
- docs: add w/ glossary and model
- test: fix sample test
- test: fix remaining
- chore: delete comments
- fix: fix package name
- chore: update version to latest (#63)
- fix: adjusts flaky test; deletes unneeded test (#66)
- chore(deps): update dependency google-cloud-vision to v2 (#71)
- chore(deps): update dependency google-cloud-texttospeech to v2.2.0 (#49)
- chore(deps): update dependency google-cloud-storage to v1.31.2 (#55)
- chore(deps): update dependency google-cloud-translate to v3.0.1 (#45)
- chore(deps): update dependency google-cloud-automl to v2 (#67)
- chore(deps): update dependency google-cloud-storage to v1.32.0 (#72)
- fix: re-generated v3 client and fix system.py (#76)
- chore(deps): update dependency google-cloud-automl to v2.1.0 (#77)
- chore: fix flaky tests (#78)
- chore(deps): update dependency google-cloud-storage to v1.33.0 (#79)
- fix: moves import six inside of region tags (#83)
- chore(deps): update dependency google-cloud-storage to v1.34.0 (#85)
- chore(deps): update dependency google-cloud-translate to v3.0.2 (#84)
- chore(deps): update dependency google-cloud-storage to v1.35.0 (#86)
- chore: update testing and docs config (#89)
- chore: fixed wrong method name (#100)
- fix: moves region tags (#103)
- chore(deps): update dependency google-cloud-vision to v2.1.0 (#110)
- chore(deps): update dependency google-cloud-storage to v1.35.1 (#111)
- chore(deps): update dependency google-cloud-vision to v2.2.0 (#113)
- chore(deps): update dependency google-cloud-storage to v1.36.0 (#112)
- chore(deps): update dependency google-cloud-storage to v1.36.1 (#114)
- chore(deps): update dependency google-cloud-automl to v2.2.0 (#117)
- chore(deps): update dependency google-cloud-storage to v1.36.2 (#118)
- chore: renamed output dir to prevent bucket concurrency error (#120)
- feat(v3beta1): add online and batch document translation  (#121)
- chore(deps): update dependency google-cloud-storage to v1.37.0 (#124)
- chore(deps): update dependency google-cloud-vision to v2.3.0 (#129)
- chore(deps): update dependency google-cloud-translate to v3.1.0 (#128)
- chore(deps): update dependency google-cloud-texttospeech to v2.3.0 (#127)
- chore(deps): update dependency google-cloud-storage to v1.37.1 (#131)
- chore: migrate to owl bot (#137)
- chore(deps): update dependency google-cloud-vision to v2.3.1 (#134)
- chore(deps): update dependency pytest to v6 (#138)
- chore(deps): update dependency google-cloud-automl to v2.3.0 (#136)
- chore(deps): update dependency google-cloud-storage to v1.38.0 (#140)
- chore(deps): update dependency pytest to v6.2.4 (#150)
- chore: refactored all glossaries samples to use predefined glossary i… (#159)
- chore: new owl bot post processor docker image (#163)
- docs: connect Python Translate client library to nebulous serverless example (#162)
- chore(deps): update dependency google-cloud-translate to v3.2.0 (#167)
- chore(deps): update dependency google-cloud-texttospeech to v2.4.0 (#166)
- samples: changed v1beta1 AutoML model for v1 (#168)
- chore(deps): update dependency google-cloud-vision to v2.3.2 (#174)
- chore(deps): update dependency google-cloud-translate to v3.2.1 (#181)
- chore(deps): update dependency google-cloud-storage to v1.39.0 (#182)
- add: Document Translation snippets - translate_document and batch_translate_document (#183)
- chore(deps): update dependency google-cloud-storage to v1.40.0 (#187)
- chore(deps): update dependency google-cloud-texttospeech to v2.5.0 (#188)
- chore(deps): update dependency backoff to v1.11.0 (#192)
- chore(deps): update dependency google-cloud-automl to v2.4.0 (#193)
- chore(deps): update dependency google-cloud-storage to v1.41.0 (#198)
- chore(deps): update dependency backoff to v1.11.1 (#200)
- chore(deps): update dependency google-cloud-vision to v2.4.0 (#194)
- chore(deps): update dependency google-cloud-storage to v1.41.1 (#208)
- chore(deps): update dependency google-cloud-translate to v3.3.1 (#212)
- feat: add Samples section to CONTRIBUTING.rst (#206)
- chore(deps): update dependency google-cloud-texttospeech to v2.5.1 (#210)
- chore(deps): update dependency google-cloud-automl to v2.4.1 (#209)
- chore(deps): update dependency google-cloud-vision to v2.4.1 (#211)
- chore(deps): update dependency google-cloud-texttospeech to v2.5.2 (#217)
- chore(deps): update dependency google-cloud-automl to v2.4.2 (#216)
- chore(deps): update dependency google-cloud-vision to v2.4.2 (#221)
- chore(deps): update dependency google-cloud-translate to v3.3.2 (#220)
- chore: fix INSTALL_LIBRARY_FROM_SOURCE in noxfile.py (#226)
- chore(deps): update dependency google-cloud-storage to v1.42.0 (#228)
- chore: drop mention of Python 2.7 from templates (#229)
- chore: Increase Batch Translate Text timeout (#225)
- chore: fix broken link in sample (#223)
- chore(deps): update dependency google-cloud-translate to v3.4.0 (#241)
- chore(deps): update dependency pytest to v6.2.5 (#237)
- chore(deps): update dependency google-cloud-storage to v1.42.1 (#248)
- chore(deps): update dependency google-cloud-storage to v1.42.2 (#250)
- chore: blacken samples noxfile template (#249)
- chore: drop six (#253)
- chore(deps): update dependency google-cloud-vision to v2.4.3 (#257)
- chore(deps): update dependency google-cloud-texttospeech to v2.5.3 (#258)
- chore(deps): update dependency google-cloud-vision to v2.4.4 (#262)
- chore: fail samples nox session if python version is missing (#261)
- chore(deps): update dependency google-cloud-translate to v3.4.1 (#263)
- chore(deps): update dependency google-cloud-storage to v1.42.3 (#264)
- chore(python): Add kokoro configs for python 3.10 samples testing (#272)
- chore(deps): update dependency google-cloud-texttospeech to v2.6.0 (#269)
- chore(deps): update dependency google-cloud-automl to v2.5.0 (#273)
- Fix: #254 by increasing timeout, use backoff module instead of flaky (#271)
- chore(deps): update all dependencies (#274)
- chore(deps): update dependency google-cloud-translate to v3.6.0 (#281)
- docs(samples): Add Cloud Code tags for API Explorer pilot (#282)
- chore(deps): update all dependencies (#283)
- chore(deps): update all dependencies (#291)
- chore(deps): update dependency google-cloud-automl to v2.5.2 (#292)
- chore(deps): update all dependencies (#293)
- chore(deps): update dependency google-cloud-texttospeech to v2.8.0 (#299)
- chore(deps): update all dependencies (#301)
- fix: 290 added a create glossary line before each call using bistro-glossary (#302)
- chore: update python-docs-samples link to main branch (#304)
- chore: updated translate_v3beta1_translate_document sample (#308)
- chore(deps): update all dependencies (#307)
- chore(python): fix undefined name 'glob' in samples noxfile (#314)
- chore(deps): update dependency google-cloud-storage to v2 (#318)
- chore(deps): update dependency google-cloud-automl to v2.6.0 (#322)
- chore(python): Noxfile recognizes that tests can live in a folder (#323)
- chore: add samples/cloud-nebulous-serverless/README.md (#312)
- chore(deps): update dependency google-cloud-texttospeech to v2.9.1 (#324)
- chore(deps): update all dependencies (#325)
- docs: add generated snippets (#336)
- chore: update copyright year to 2022 (#339)
- chore(deps): update all dependencies (#335)
- chore(deps): update all dependencies (#342)
- chore: Adding support for pytest-xdist and pytest-parallel (#349)
- chore(deps): update all dependencies (#352)
- chore(deps): update dependency google-cloud-texttospeech to v2.11.0 (#355)
- chore(deps): update dependency pytest to v7.1.0 (#356)
- chore(deps): update dependency google-cloud-storage to v2.2.0 (#357)
- chore(deps): update dependency google-cloud-storage to v2.2.1 (#358)
- chore(deps): update dependency pytest to v7.1.1 (#359)
- chore(python): use black==22.3.0 (#363)
- chore(deps): update dependency google-cloud-vision to v2.7.2 (#362)
- chore(deps): update dependency google-cloud-storage to v2.3.0 (#378)
- chore: use gapic-generator-python 0.65.1 (#377)
- chore(python): add nox session to sort python imports (#381)
- chore(deps): update dependency pytest to v7.1.2 (#385)
- chore(deps): update dependency backoff to v2 (#386)
- chore(deps): update dependency backoff to v2.0.1 (#388)
- chore(deps): update dependency google-cloud-translate to v3.7.3 (#392)
- fix: require python 3.7+ (#411)
- chore(deps): update all dependencies (#400)
- chore: resolve issue with prerelease presubmit [autoapprove] (#417)
- chore(deps): update all dependencies (#420)
- chore(deps): update all dependencies (#422)
- chore(deps): update all dependencies (#423)
- chore(deps): update all dependencies (#428)
- docs(samples): fixing file saving for local documents translation (#419)
- chore(deps): update dependency pytest to v7.1.3 (#443)
- chore: Bump gapic-generator-python version to 1.3.0 (#445)
- chore(deps): update dependency google-cloud-translate to v3.8.2 (#444)
- chore: detect samples tests in nested directories (#448)
- chore(deps): update dependency google-cloud-vision to v3.1.2 (#450)
- chore(deps): update dependency google-cloud-vision to v3.1.3 (#453)
- chore(deps): update all dependencies (#455)
- chore(deps): update dependency backoff to v2.2.1 (#456)
- chore(deps): update all dependencies (#459)
- chore(deps): update dependency pytest to v7.2.0 (#460)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by telpirion** [comment]
Set this to "Ready for Review" too soon -- still need to remove old / unused samples.

File: translation/samples/generated_samples/snippet_metadata_translate_v3.json [hunk 1]
```diff
@@ -0,0 +1,1728 @@
+{
```
Comment (dandhlee) [inline translation/samples/generated_samples/snippet_metadata_translate_v3.json:1]: Snippetgen samples are not in scope, please remove this directory and keep it in the old repository.

File: translation/samples/snippets/noxfile.py [hunk 2]
```diff
@@ -0,0 +1,312 @@
+# Copyright 2019 Google LLC
```
Comment (dandhlee) [inline translation/samples/snippets/noxfile.py:1]: Please remove this file.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Could you rename the directory to be `translate`? This change was made to rename from `python-translation` to `python-translate`.

**Comment by telpirion** [comment]
> Could you rename the directory to be `translate`? This change was made to rename from `python-translation` to `python-translate`.

The name of the product is Cloud Translation. See: https://cloud.google.com/translate/docs/

File: translation/samples/generated_samples/snippet_metadata_translate_v3.json (hunk 1 above)
Comment (telpirion) [reply translation/samples/generated_samples/snippet_metadata_translate_v3.json]: Done.

File: translation/samples/snippets/noxfile.py (hunk 2 above)
Comment (telpirion) [reply translation/samples/snippets/noxfile.py]: Done.

**Comment by dandhlee** [comment]
The name of the product is indeed `Translation` but the shortname is `translate`. Indeed strange, but I'd prefer to be consistent.

**Review by dandhlee — APPROVED** [review]
LGTM. Overriding CLA and merging.

**Pull request merged**